src/Illuminate/Database/Query/Grammars/Grammar.php

```diff
@@ -612,6 +612,20 @@ protected function compileUnion(array $union)
     }
 
     /**
+     * Compile an exists statement into SQL.
+     *
+     * @param \Illuminate\Database\Query\Builder $query
+     *
+     * @return string
+     */
+    public function compileExists(Builder $query)
+    {
+        $select = $this->compileSelect($query);
```

Comment:
Via https://dev.mysql.com/doc/refman/5.0/en/exists-and-not-exists-subqueries.html:
> Traditionally, an EXISTS subquery starts with SELECT *, but it could begin with SELECT 5 or SELECT column1 or anything at all. MySQL ignores the SELECT list in such a subquery, so it makes no difference.
Huh, TIL